static/src/stylesheets/module/identity/_membership-tab.scss

```diff
@@ -0,0 +1,272 @@
+/* Edit profile Membership tab
+   ========================================================================== */
+.membership-tab {
+    padding-bottom: $gs-baseline*2;
+
+    .show-updating {
+        display: inline-block;
+    }
+
+    .membership-loader {
```

Comment:
Feels like a slightly awkward level of nesting here, and only slightly using BEM classnames, but it seems similar to the rest of the CSS so probably not worth changing too much in this PR.